### DISCUSSION TITLE
Date picker truncated on mobile page

### DISCUSSION THREAD

**Discussion by syeillah** [Budibase/budibase#4561] [Help] [answered]
hi, so im using date picker for selecting estimation date and it works on desktop page. 

<img width="570" alt="Screen Shot 2022-02-18 at 13 58 00" src="https://user-images.githubusercontent.com/99949174/154633482-ad29fc4c-1c97-491c-bbc8-54712a5f73c4.png">

But, when it comes to mobile page, the date picker become truncated like this:

<img width="507" alt="Screen Shot 2022-02-18 at 13 58 17" src="https://user-images.githubusercontent.com/99949174/154633544-5befbe18-49e3-4884-a82f-45f83f307af1.png">

May i know the solution for this? since the app itself will be used for mobile. 
Thank you!

**Comment by mjashanks** [comment, marked as answer]
Thanks for reporting this. I can't see an obvious workaround - but have logged this as an issue: https://github.com/Budibase/budibase/issues/4562

This actually works for me, when testing in responsive mode in Chrome. What browser & device are you using to test this on?

**Reply by syeillah** [reply]
Hi, thanks for the answer!
Actually i also using Chrome (my device is mac).. but no worries I just tried checking through a mobile browser and it doesn't get cut. 
Thank you.
